src/Menu/MenuItem.tsx

```diff
@@ -1,6 +1,10 @@
-import React from 'react';
+import React, { HTMLAttributes } from 'react';
 import { StyledMenuItemWrapper } from './MenuItem.styles';
 
-export const MenuItemWrapper: React.FC = ({ children }) => {
-  return <StyledMenuItemWrapper>{children}</StyledMenuItemWrapper>;
+export interface MenuItemProps extends HTMLAttributes<HTMLDivElement> {
+  appName?: 'maps' | 'occupiers' | 'insight' | 'helpCentre';
```

Comment:
```suggestion
  appName?: 'maps' | 'occupiers' | 'insight';
```

Don't think we want to add help centre here as it will always lie outside our react apps.

Reply:
We have it on the logo component because it will be needed for the dropdowns and dashboard.